application/models/Template.php

```diff
@@ -293,6 +293,10 @@ public static function getTemplatePath($sTemplateName = "")
         }
 
         $oTemplate = self::model()->findByPk($sTemplateName);
+        if (empty($oTemplate))
+        {
+            throw new \Exception("Survey theme {$sTemplateName} not found.", 1);
```

Comment:
Not required, but if you want to avoid `\`, you can add `use Exception;` at top of file (if there's a namespace, not sure there is right now).

Reply:
Something to discuss is which exception messages should be translated with gT().

Reply:
Must use CException since we use Yii (and there are no namespace)

Reply:
OK, sure.

Reply:
Fixed